driver-http/src/main/resources/activities/documents-api/http-docsapi-search-advanced.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Documents API Search Advanced
+weight: 2
+---
+
+## Description
+
+The Documents API Search Advanced workflow targets Stargate's Documents API using generated JSON documents. Specifically, it looks to benchmark more advanced search cases, using both complex boolean logic and filters that aren't natively supported by the underlying data store.
+By default, the documents used are sharing the same structure and are approximately half a kilobyte in size each:
+
+* Each document has 15 leaf values, with a maximum depth of 3
+* there is at least one `string`, `boolean`, `number` and `null` leaf
+* there is one array with `double` values and one with `string` values
+* there is one empty array and one empty map
+
+The example JSON looks like:
+
+```json
+{
+  "user_id":"56fd76f6-081d-401a-85eb-b1d9e5bba058",
+  "created_on":1476743286,
+  "full_name":"Andrew Daniels",
+  "married":true,
+  "address":{
+    "primary":{
+      "cc":"IO",
+      "city":"Okmulgee"
+    },
+    "secondary":{
+
+    }
+  },
+  "coordinates":[
+    64.65964627052323,
+    -122.35334535072856
+  ],
+  "children":[
+
+  ],
+  "friends":[
+    "3df498b1-9568-4584-96fd-76f6081da01a"
+  ],
+  "debt":null,
+  "match1": 0, // or 1000
+  "match2": "true", // or "false"
+  "match3": true // or false
+}
+```
+
+## Filters tested
+
+The advanced search workload can test the following `where` clauses:
+- in: match1 IN [0]
```

Comment:
can we have expression in the `` looks nicer and more readable:

* in: `match1 IN [0]`

(same for the other readme)